fetch/src/3_visualize.r

```diff
@@ -0,0 +1,45 @@
+plot_data <- function() {
+  # Create a plot
+  png(file = file.path(project_output_dir, 'figure_1.png'), width = 8, height = 10, res = 200, units = 'in')
+  par(omi = c(0,0,0.05,0.05), mai = c(1,1,0,0), las = 1, mgp = c(2,.5,0), cex = 1.5)
+  
+  plot(NA, NA, xlim = c(2, 1000), ylim = c(4.7, 0.75),
+       ylab = "Test RMSE (°C)", xlab = "Training temperature profiles (#)", log = 'x', axes = FALSE)
+  
+  n_profs <- c(2, 10, 50, 100, 500, 980)
+  
+  axis(1, at = c(-100, n_profs, 1e10), labels = c("", n_profs, ""), tck = -0.01)
+  axis(2, at = seq(0,10), las = 1, tck = -0.01)
+  
+  # slight horizontal offsets so the markers don't overlap:
+  offsets <- data.frame(pgdl = c(0.15, 0.5, 3, 7, 20, 30)) %>%
+    mutate(dl = -pgdl, pb = 0, n_prof = n_profs)
+  
+  for (mod in c('pb','dl','pgdl')){
+    eval_data <- file.path(project_output_dir, 'model_summary_results.csv')
```

Comment:
yet here, you'll want to read in from what will now be `2_process/out`.

Reply:
Just to confirm, this should read in the 'model_summary_results.csv' correct?

Reply:
Yes - you'll need to read the csv in, rather than just specify the file path. Also - as written, the file is getting read in on every iteration of the loop. Since the `eval_data` object is not specific to each iteration of the loop, it will be more efficient to read in `eval_data` once, before the for loop, and then simply filter it within the loop.